test/testdata/rbi/big_math.rb

```diff
@@ -0,0 +1,53 @@
+# typed: true
+
+require "bigdecimal/math"
+
+class TestWrapper
+  include BigMath
+
+  def my_e(prec)
+    E(prec)
+  end
+
+  def my_pi(prec)
+    PI(prec)
+  end
+
+  def my_atan(x, prec)
+    atan(x, prec)
+  end
+
+  def my_cos(x, prec)
+    cos(x, prec)
+  end
+
+  def my_sin(x, prec)
+    sin(x, prec)
+  end
+
+  def my_sqrt(x, prec)
+    sqrt(x, prec)
+  end
+end
+
+BigMath.exp(BigDecimal("4"), 2)
+BigMath.log(BigDecimal("4"), 2)
```

Comment:
It seems `log` and `exp` are not part of the module_function so didn't update them above but added a test case for them.